components/script_bindings/codegen/codegen.py

```diff
@@ -5643,7 +5685,7 @@ def __init__(self, name, returnType, args, inline=False, static=False,
         self.unsafe = unsafe
         ClassItem.__init__(self, name, visibility)
 
-    def getDecorators(self, declaring):
+    def getDecorators(self, declaring) -> str:
```

Comment:
```suggestion
    def getDecorators(self, declaring: bool) -> str:
```

The type is missing. I guess it is a bool, but not sure.